packages/node_modules/samples/browser-single-party-call/index.html

```diff
@@ -57,7 +57,7 @@ <h1>Single Party Calling</h1>
     </div>
 
     <!-- /bundle.js is the bundled Webex JavaScript SDK -->
-    <script src="/bundle.js"></script>
+    <script src="/meetings.bundle.js"></script>
```

Comment:
Instead of doing `meetings.bundle.js`, we need to get rid of it.
Make `meetings.bundle.js` --> `bundle.js`

Reply:
I can make a git patch and you can apply it.
Probably easier than writing out what needs to be replaced